### PR TITLE
[#3579] fix(license): mention license for svg files

### PR DESCRIPTION
**Pull request by LauraXia123**
### What changes were proposed in this pull request?
mention license for svg files

### Why are the changes needed?
mention license for svg files

Fix: #3579

### Does this PR introduce _any_ user-facing change?
N/A

### How was this patch tested?
N/A

### CODE REVIEW

**Review by justinmclean — CHANGES_REQUESTED** [review]
Again it is unclear where these come from and exactly how they are licensed. Please list the source of teh files so we can work this out.

**Comment by jerryshao** [comment]
> Again it is unclear where these come from and exactly how they are licensed. Please list the source of teh files so we can work this out.

They are already listed in the license file.

**Comment by jerryshao** [comment]
@justinmclean we don't want this to block our release, just show us exactly how to fix it, or we will leave as it is.

File: web/src/lib/icons/svg/doris.svg [hunk 1]
```diff
@@ -1,4 +1,25 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Referred from Apache Doris's icon.svg implementation
+ * https://github.com/apache/doris/blob/master/.idea/icon.svg
+-->
```
Comment (jerryshao) [inline web/src/lib/icons/svg/doris.svg:21]: @justinmclean  please check here.

**Comment by jerryshao** [comment]
Talked with @LauraXia123 offline, we have already listed the source in the SVG file, you can check the source diffs of the these svg files. @justinmclean

**Comment by jerryshao** [comment]
```
 *
 * Referred from Apache Doris's icon.svg implementation
 * https://github.com/apache/doris/blob/master/.idea/icon.svg
-->
```

**Comment by justinmclean** [comment]
The short version is that it is not clear how this is licensed and that content on a website doesn't have to be compatible with the ASF license as it has not gone through a release process. We can also not add or alter headers of files not created by us, to do that you need the owners permission.

**Comment by justinmclean** [comment]
As discussed yesterday, I'll take a look at this today and propose a better solution.

**Pull request closed without merging**